__tests__/index.test.js

```diff
@@ -107,3 +107,62 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 });
+describe("GET /api/articles", () => {
+  it("GET 200: should serve an array of all articles, where each has the following attributes: author, title, article_id, topic, created_at, votes, article_img_url, comment_count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const {articles} = body
+        console.log(articles[2]);
+        const actual = articles[2];
+        const expected = {
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: '2020-08-03T13:14:00.000Z',
+          votes: '17',
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: '2',
+        };
+        expect(actual).toMatchObject(expected);
+      });
+  });
+  it("they need to be sorted by created_at date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const {articles} = body
+        const expectedSortBy = "created_at"
+        const expectedOrder = {descending: true, coerce: true}
+        expect(articles).toBeSortedBy(expectedSortBy,expectedOrder);
+      });
+  });
+  it("there should not be a body property present on any of the article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const {articles} = body
+        const actual = articles[2];
+        const expected = {
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          created_at: '2020-08-03T13:14:00.000Z',
+          votes: '17',
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: '2',
+        };
+        expect(actual).toMatchObject(expected);
```

Comment:
This test would pass if there was a body property due to how `.toMatchObject` works - how else can we check the an object does not have a property?
It's also more thorough to loop through the whole array of articles rather than just look at one